deployment/environment/devenv/don.go

```diff
@@ -478,7 +478,7 @@ func (n *Node) SetUpAndLinkJobDistributor(ctx context.Context, jd JobDistributor
 	// now create the job distributor in the node
 	id, err := n.CreateJobDistributor(ctx, jd)
 	if err != nil &&
-		(!strings.Contains(err.Error(), "only a single feeds manager is supported") || !strings.Contains(err.Error(), "DuplicateFeedsManagerError")) {
+		!strings.Contains(err.Error(), "DuplicateFeedsManagerError") {
```

Comment:
only a single feeds manager is supported is a valid error and should not excluded